docs/hardware/devices/seeed-studio/sensecap/card-tracker.mdx

```diff
@@ -9,6 +9,8 @@ SenseCAP T1000-E is a high-performance tracker designed for Meshtastic. As small
 
 :::warning[Note]
 Currently, LR1110 radios are unable to receive Meshtastic packets from the older SX127x radios. Semtech is aware of this issue, and we're hopeful they'll provide a solution soon.
+Currently, this device is not supported in Meshtastic. There is no way to use the tracker functions.
```

Comment:
Where did you find this information @mspohr? I have heard of many people using the T1000 successfully